substrate/frame/staking-async/src/pallet/mod.rs

```diff
@@ -620,8 +620,7 @@ pub mod pallet {
 	impl<T: Config> Get<u32> for ClaimedRewardsBound<T> {
 		fn get() -> u32 {
 			let max_total_nominators_per_validator =
-				<T::ElectionProvider as ElectionProvider>::MaxBackersPerWinner::get() *
-					<T::ElectionProvider as ElectionProvider>::Pages::get();
+				<T::ElectionProvider as ElectionProvider>::MaxBackersPerWinnerFinal::get();
```

Comment:
@Ank4n this was a subtle mistake of the past that I put in. MaxWinnersPerPage * Pages is also not incorrect, but might be an overestimation. Now that I expose `MaxBackersPerWinnerFinal`, we can use this.